src/cloudimagedirectory/transform/transform.py

```diff
@@ -564,3 +564,50 @@ def run(self, data: type[Transformer]) -> list:
                     connection.DataEntry(f"v2/os/{os}/provider/{provider}/version/list", version_map[provider])
                 )
         return results
+
+
+class TransformerV2ListRegionByVersion(Transformer):
+    """Generate a list for all available regions for one version."""
+
+    def run(self, data):
```

Comment:
Add `# noqa: C901` here to silence the complexity linter. It would be good to revisit how we could make this less complex later. :wink:

Reply:
Ah right, I added it :)